service.py

```diff
@@ -5,20 +5,17 @@
 from sys import stderr
 from urllib.parse import parse_qs, urlparse
 from yt_dlp import YoutubeDL
+import json
 
 class Handler(BaseHTTPRequestHandler):
     error_message_format = '%(explain)s'
 
     def __init__(self, *args, **kwargs):
```

Comment:
Is this now redundant, because it doesn't do anything other than call the super constructor?

Reply:
Yep. Removed the constructor and `error_message_format`. I can't find a point in the history where `error_message_format` was ever used.